lib/feature/start/presentation/pages/start_page.dart

```diff
@@ -1,207 +1,109 @@
-                          }
-                        },
+            BlocBuilder<StartNextBloc, StartState>(
+              builder: (context, state) {
+                if (state is LoadedStartState) {
+                  return Container(
+                    decoration: BoxDecoration(
+                      border: Border.all(color: Colors.black54, width: 2),
+                      borderRadius: BorderRadius.circular(8),
+                    ),
+                    child: Padding(
+                      padding: const EdgeInsets.all(10.0),
+                      child: Column(
+                        children: [
+                          Text(
+                              "${state.numberOfAnsweringQuestions}/${state.numberOfTotalQuestions}"),
+                          Text(
+                            state.nowQuestion,
+                            style: TextStyle(fontSize: 20),
+                          ),
+                        ],
                       ),
-                    },
-                    child: Text('Next'),
-                  ),
-                ],
-              ),
-            ),
-          ),
-        ],
-      ),
-    );
-  }
-}
-
-class BodyContainer extends StatelessWidget {
-  @override
-  Widget build(BuildContext context) {
-    return Container(
-      decoration: BoxDecoration(
-        border: Border.all(color: Colors.black54, width: 2),
-        borderRadius: BorderRadius.circular(8),
-      ),
-      child: StreamBuilder<QuerySnapshot>(
-        stream: FirebaseFirestore.instance.collection('questions').snapshots(),
-        builder: (context, snapshot) {
-          return (!snapshot.hasData)
-              ? Container()
-              : ListView.builder(
-                  itemCount: snapshot.data.docs.length,
-                  itemBuilder: (context, index) {
-                    return Column(
-                      children: [
-                        QuestionCard(
-                          problemInt: index + 1.toInt(),
-                          questionText:
-                              snapshot.data.docs[index].get('questionText'),
-                        ),
-                      ],
-                    );
-                  },
+                    ),
+                  );
+                } else if (state is FinishQuestionState) {
+                  return Column(
+                    children: [
+                      Text(
+                        "finished !",
+                        style: TextStyle(fontSize: 30),
+                      ),
+                      RaisedButton(
+                        onPressed: Navigator.of(context).pop,
+                        child: Text("Top"),
+                      )
+                    ],
+                  );
+                }
+                return Center(
+                  child: CircularProgressIndicator(),
                 );
-        },
-      ),
-    );
-  }
-}
-
-class QuestionCard extends StatelessWidget {
```

Comment:
This QuestionCard class is useless part that should be removed at earlier stage😕